ynr/apps/people/models.py

```diff
@@ -297,8 +297,19 @@ def record_version(self, change_metadata, new_person=False):
         new_version["data"] = get_person_as_version_data(
             self, new_person=new_person
         )
-        if not versions or new_version["data"] != versions[0]["data"]:
+        should_insert = True
+
+        if versions and new_version["data"] == versions[0]["data"]:
+            # Don't create empty versions
+            should_insert = False
+
+        if new_version["information_source"].startswith("After merging person"):
```

Comment:
More generally, it would be nice if we could identify merges with a flag instead of relying on a substring in the source. Just leaving a comment here so I can reference it from an issue.